### PR TITLE
Testing threading for comments

### PR DESCRIPTION
**Pull request by amogh-rokt**

### CODE REVIEW

**Comment by amogh-rokt** [comment]
testing

**Review by moq4 — COMMENTED** [review]
ok

**Comment by moq4** [comment]
test

**Review by moq4 — COMMENTED** [review]
sdf

**Review by moq4 — COMMENTED** [review]
asd

**Review by moq4 — COMMENTED** [review]
review

**Review by moq4 — COMMENTED** [review]
sdf

**Review by moq4 — COMMENTED** [review]
pls

**Review by moq4 — COMMENTED** [review]
ok

File: index.js [hunk 1]
```diff
@@ -29,13 +29,16 @@ try {
   let webhookUrl = core.getInput("webhook");
   webhookUrl += "&messageReplyOption=REPLY_MESSAGE_FALLBACK_TO_NEW_THREAD";
 
+  const threadKey = github.context.payload.repository.name + github.context.ref;
+
+  console.log(github);
```
Comment (moq4) [inline index.js:74]: good

File: index.js [hunk 2]
```diff
@@ -29,13 +29,17 @@ try {
   let webhookUrl = core.getInput("webhook");
   webhookUrl += "&messageReplyOption=REPLY_MESSAGE_FALLBACK_TO_NEW_THREAD";
 
+  const threadKey = github.context.payload.repository.name + github.context.ref;
+
```
Comment (moq4) [inline index.js:73]: ya

**Review by moq4 — CHANGES_REQUESTED** [review]
wtf

**Review by moq4 — APPROVED** [review]
lgtm

**Review by moq4 — APPROVED** [review]
lgtm

**Review by moq4 — COMMENTED** [review]
sdfsdf

**Review by moq4 — CHANGES_REQUESTED** [review]
wtf

**Review by moq4 — COMMENTED** [review]
sdf

**Review by moq4 — CHANGES_REQUESTED** [review]
wtf

**Review by moq4 — APPROVED** [review]
lgtm!

**Comment by jacobsalway** [comment]
nice

**Pull request merged**